torchgeo/datasets/gid15.py

```diff
@@ -0,0 +1,237 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""GID-15 dataset."""
+
+import glob
+import os
+from typing import Callable, Dict, List, Optional
+
+import numpy as np
+import torch
+from PIL import Image
+from torch import Tensor
+
+from .geo import VisionDataset
+from .utils import download_and_extract_archive
+
+
+class GID15(VisionDataset):
+    """GID-15 dataset.
+
+    The `GID-15 <https://captain-whu.github.io/GID15/>`_
+    dataset is a dataset for semantic segmentation.
+
+    Dataset features:
+    * images taken by the Gaofen-2 (GF-2) satellite over 60 cities in China
+    * masks representing 15 semantic categories
+    * three spectral bands - RGB
+    * 150 with 3 m per pixel resolution (6800x7200 px)
+
+    Dataset format:
+    * images are three-channel pngs
+    * masks are single-channel pngs
+    * colormapped masks are 3 channel tifs
+
+    Dataset classes:
+        1. background
+        2. industrial_land
+        3. urban_residential
+        4. rural_residential
+        5. traffic_land
+        6. paddy_field
+        7. irrigated_land
+        8. dry_cropland
+        9. garden_plot
+        10. arbor_woodland
+        11. shrub_land
+        12. natural_grassland
+        13. artificial_grassland
+        14. river
+        15. lake
+        16. pond
```

Comment:
I would remove the extra indentation from these lines

Reply:
Fixed